models/cluster.x-k8s.io.machine.js

```diff
@@ -49,6 +58,28 @@ export default class CapiMachine extends SteveModel {
     downloadUrl(this.links.sshkeys);
   }
 
+  toggleForceRemoveModal(resources = this) {
+    this.$dispatch('promptModal', {
+      resources,
+      component: 'ForceMachineRemoveDialog'
+    });
+  }
+
+  async forceMachineRemove() {
+    const machine = await this.machineRef();
+
+    machine.setAnnotation('provisioning.cattle.io/force-machine-remove', 'true');
```

Comment:
There's a file that contains a list of annotations, this should probably be listed there (`./config/labels-annotations.js` `CAPI` obj)